src/index.js

```diff
@@ -209,6 +216,10 @@ export class BrowserHistory extends History {
       this.options.routeHandler(fragment) :
       false;
   }
+
+  _createOrigin(protocol: string, hostname: string, port: string) {
```

Comment:
Yes. I originally had pal-browser for twsting but soon realized it was
brittle in this case. We should remove the reference. As for the helper
method, for it to be testable from an import it would have to be an
exported function, right?
On Feb 18, 2016 1:29 PM, "Bryan Smith" <notifications@github.com> wrote:

> In src/index.js
> <https://github.com/aurelia/history-browser/pull/22#discussion_r53358316>:
>
> > @@ -209,6 +216,10 @@ export class BrowserHistory extends History {
> >        this.options.routeHandler(fragment) :
> >        false;
> >    }
> > +
> > +  _createOrigin(protocol: string, hostname: string, port: string) {
>
> This could be a regular helper function instead of a class method.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aurelia/history-browser/pull/22/files#r53358316>.
>